Formula/lefthook.rb

```diff
@@ -0,0 +1,21 @@
+class Lefthook < Formula
+  desc "Fast and powerful Git hooks manager for any type of projects"
+  homepage "https://github.com/evilmartians/lefthook"
+  url "https://github.com/evilmartians/lefthook/archive/refs/tags/v0.7.3.tar.gz"
+  sha256 "643bb3d2a1b67d865285a9cadcad11f2588ba284570ab9924c8e3d2e902301ca"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args
+  end
+
+  test do
+    mkdir_p testpath/".git/hooks"
+    system bin/"lefthook install"
+
+    assert_predicate testpath/"lefthook.yml", :exist?
+    assert_match "#{version}", shell_output("#{bin}/lefthook version").strip
```

Comment:
This shouldn't be needed. `assert_match` also allows partial matches
```suggestion
    assert_match "#{version}", shell_output("#{bin}/lefthook version")
```